src/test/resources/features/complaints/ComplaintsEndToEnd.feature

```diff
@@ -77,6 +99,16 @@ Feature: Complaints End To End
       | Ex-Gratia        |
       | Minor Misconduct |
 
+  Scenario Outline: User moves a BF case to the QA stage
+    Given I am logged into "CS" as user "BF_USER"
+    When I create a "BF" case and move it to the "<complaintType> QA" stage
```

Comment:
and again here. iCreateACOMPCaseForAComplaintAndMoveItToStage allows for specifying the complaint type and the stage